src/scrabble.js

```diff
@@ -1,6 +1,48 @@
+const val1 = ["A", "E","I","O","U","L","N","R","S","T","a","e","i","o","u","l","n","r","s","t",];
+const val2 = ["D", "d", "G", "g"];
+const val3 = ["B", "b", "C", "c", "M", "m", "P", "p"];
+const val4 = ["F", "f", "H", "h", "V", "v", "W", "w", "Y", "y"];
+const val5 = ["K", "k"];
+const val8 = ["J", "j", "X", "x"];
+const val10 = ["Q", "q", "Z", "z"];
+
 class Scrabble {
   // Write your implementation here
 
+  constructor(word) {
+    this.word = word;
+   }
+
+  score() {
+    let score = 0
+
+    if (this.word === undefined || this.word === null) {
+      return 0
+    }
```

Comment:
in here you could then handle lower case words like:
let word = this.word.trim().toUpperCase();